packages/core/src/__mocks__/sign-in-experience.ts

```diff
@@ -30,6 +32,15 @@ export const mockSignInExperience: SignInExperience = {
     fallbackLanguage: 'en',
     fixedLanguage: 'zh-CN',
   },
+  signUp: {
+    identifier: SignUpIdentifier.Username,
+    password: true,
+    verify: false,
+  },
+  signIn: {
+    methods: [{ identifier: SignInIdentifier.Username, password: true, verificationCode: false }],
+    isPasswordPrimary: true,
+  },
```

Comment:
should submit with the migration script?

Reply:
Well, will create another PR merging to [feature/sie-v2](https://github.com/logto-io/logto/tree/feature/sie-v2)